### PR TITLE
Ylp 1445 add alerts in patients table

### PR DESCRIPTION
**Pull request by TimCanu**

### CODE REVIEW

File: packages/yourloops/pages/hcp/patients/table.tsx [hunk 1]
```diff
@@ -178,22 +211,49 @@ function PatientTable(props: PatientTableProps): JSX.Element {
...
                   id={`patients-list-header-alert-time-target${orderBy === PatientTableSortFields.alertTimeTarget ? `-${order}` : ""}`}
                   active={orderBy === PatientTableSortFields.alertTimeTarget}
                   direction={order}
                   onClick={createSortHandler(PatientTableSortFields.alertTimeTarget)}>
-                  {t("alert-time-target")}
+                  {t("time-out-of-range-target")}
+                  <StyledTooltip
+                    title={
+                      <Typography color="inherit">{t("time-out-of-range-target-tooltip")}</Typography>
+                    } arrow>
```
Comment (fatb38) [inline packages/yourloops/pages/hcp/patients/table.tsx:224]: better readability
```suggestion
                   <StyledTooltip
                    title={<Typography color="inherit">{t("time-out-of-range-target-tooltip")}</Typography>}
                    arrow
                  >
```

File: packages/yourloops/pages/hcp/patients/table.tsx [hunk 2]
```diff
@@ -178,22 +211,49 @@ function PatientTable(props: PatientTableProps): JSX.Element {
                 </TableSortLabel>
               </StyledTableCell>
               <StyledTableCell id="patients-list-header-alert-time-target"
-                className={`${classes.tableCellHeader} ${classes.alertTimeTargetHeader}`}>
+                className={`${classes.tableCellHeader} ${classes.alertTimeTargetHeader} ${patientListCommonClasses.mediumCell}`}>
```
Comment (fatb38) [inline packages/yourloops/pages/hcp/patients/table.tsx:214]: better readability
```suggestion
               <StyledTableCell
                id="patients-list-header-alert-time-target"
                className={`${classes.tableCellHeader} ${classes.alertTimeTargetHeader} ${patientListCommonClasses.mediumCell}`}
              >
```

**Review by fatb38 — CHANGES_REQUESTED** [review]

File: packages/yourloops/pages/hcp/patients/table.tsx [hunk 3]
```diff
@@ -86,16 +95,39 @@ const patientListStyle = makeStyles(
...
+      maxWidth: "300px",
+    },
+    mediumCell: {
+      maxWidth: "200px",
+    },
+  };
+}
+);
 
 export const StyledTableCell = styled(withTheme(TableCell))((props) => ({
```
Comment (fatb38) [inline packages/yourloops/pages/hcp/patients/table.tsx:110]: could be moved to a styled component folder or something like this ?

**Review by fatb38 — CHANGES_REQUESTED** [review]

File: packages/yourloops/pages/hcp/patients/table.tsx [hunk 4]
```diff
@@ -86,16 +95,39 @@ const patientListStyle = makeStyles(
...
     height: "inherit",
   },
 }));
 
+const StyledTooltip = withStyles((theme) => ({
+  tooltip: {
+    backgroundColor: theme.palette.primary.main,
+    boxShadow: theme.shadows[1],
+    fontSize: "12px",
+    whiteSpace: "pre-line",
```
[the remark this answers is not in the document]
Comment (TimCanu) [reply packages/yourloops/pages/hcp/patients/table.tsx:124]: check the tooltip you blind boi

**Review by fatb38 — APPROVED** [review]
LGTM

**Pull request merged**